src/tasks/MonoTargetsTasks/ILStrip/ILStrip.cs

```diff
@@ -310,6 +326,9 @@ private void CreateTrimmedAssembly(PEReader peReader, string trimmedAssemblyFile
             memStream.Position = 0;
             memStream.CopyTo(os);
         }
+
+        string trimmedMethodsStr = string.Join(Environment.NewLine, trimmedMethods.ToArray());
+        File.WriteAllText(trimmedMethodListFilePath, trimmedMethodsStr);
```

Comment:
this is potentially a huge string.  It would be better to write out each element one at a time.

Reply:
Are you worried about potentially running out of memory?